Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM golang:1.14.2-alpine3.11
-RUN apk add --no-cache bash nodejs npm
+FROM centos:centos8
```

Comment:
If we really want to use non alpine versions, then shouldn't we go with fedora? It will be easier for us to test as well then. 

(I don't agree that we should change to a distro that is 10 times larger, just for journal metrics to work. IMO, we should stop journal metrics in alpine (like it does not work on Mac) and that should be the answer to the fix. Journal metrics are not that useful to worth this risk.)

Reply:
Considering the `linux/amd64` OS/Arch, the compressed size for centos8 docker image is `71.7Mb` whereas for golang-alpine version is near about `100Mb`. For reference I have linked the screenshots of the details from dockerhub below:

**Golang Alpine version**
![WhatsApp Image 2021-03-21 at 5 45 06 PM](https://user-images.githubusercontent.com/36980003/111905304-f36e6280-8a70-11eb-984f-abda7f9dd0e2.jpeg)


**CentOS**
![WhatsApp Image 2021-03-21 at 5 45 24 PM](https://user-images.githubusercontent.com/36980003/111905272-ce79ef80-8a70-11eb-8a0f-4a1ff2ef10d3.jpeg)